src/desktop/scanner.desktop

```diff
@@ -11,6 +11,7 @@ X-SuSE-YaST-AutoInst=
 X-SuSE-YaST-Geometry=
 X-SuSE-YaST-SortKey=
 X-SuSE-YaST-AutoInstResource=
+X-SuSE-YaST-Keywords=system,hardware
```

Comment:
Maybe also `photo`, `document` and even `ocr` (though it does not configure it but scanner is usually involved :wink: ).